packages/webapp/src/views/credits-purchase/credits-development-location.html

```diff
@@ -0,0 +1,89 @@
+{% extends 'credits-form-layout.html' %}
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set pageHeading = "Confirm the development details" %}
+{% block formContent %}
+  <div class="govuk-body">
+    {% if err %}
+        {% include "errorbanner.html" %}
+    {% endif %}
+    {% set idPrefix = 'confirmDevDetails' %}
+    {% set name = 'confirmDevDetails' %}
+    {% set id = 'dev-details-checked-yes' %}
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        {{ pageHeading }}
+      </h1>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+      <p>These details have been taken from the Secretary of State Metric file your local planning authority reviewed.</p>
```

Comment:
Should be `These details have been taken from the statutory biodiversity metric calculation tool your local planning authority reviewed.` to match the copy deck